### PR TITLE
Change scheme to schema

### PR DESCRIPTION
**Pull request by bedroesb**
As discussed in #729, the correct word is schema.

### CODE REVIEW

**Review by bianchini88 — CHANGES_REQUESTED** [review]
* "naming schema" looks a bit weird. Shall we have something like "naming convention"?
Or simplify and just say "consistent naming" ?
* What do we mean by "identifier schema" in the FAIRsharing entry? Also, why is this the only term which is not a plural?

File: pages/data_life_cycle/preserving.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ When preparing data for preservation several requirements need to be fulfilled.
   * Include information about provenance.
   * Include sufficient licensing information.
   * Ensure that data is well organised.
-  * Ensure that a consistent naming scheme is used.
+  * Ensure that a consistent naming schema is used.
```
Comment (nsoranzo) [inline pages/data_life_cycle/preserving.md:57]: Not sure about the others, but "naming scheme" is much more used than "naming schema" (1.540.000 Google hits vs. 55.100).

Comment (floradanna) [reply pages/data_life_cycle/preserving.md:57]: "naming convention" is what is meant here, I think

File: pages/your_domain/plant_sciences.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ It is recommended that metadata collection is contemplated from the start of the
   * [e!DAL](https://edal.ipk-gatersleben.de/) is a free data management platform for which MIAPPE templates are in development.
   * The [ISA-Tools](https://isa-tools.org/) also include a configuration for MIAPPE and can be used both for filling-in metadata and for validating.
   * [Collaborative Open Plant Omics (COPO)](https://copo-project.org/) is a data management platform specific for the plant sciences.
-  * [FAIRsharing](https://fairsharing.org) is a manually curated registry of reporting guidelines, vocabularies, identifier scheme, models, formats, repositories, knowledge bases, and data policies that includes many resources relevant for managing plant phenotyping data.
+  * [FAIRsharing](https://fairsharing.org) is a manually curated registry of reporting guidelines, vocabularies, identifier schema, models, formats, repositories, knowledge bases, and data policies that includes many resources relevant for managing plant phenotyping data.
```
Comment (floradanna) [inline pages/your_domain/plant_sciences.md:111]: "scheme" might be the correct words here. Apparently, "identification scheme" seems to be a thing.

**Comment by martin-nc** [comment]
Yes, looks like it's hard to come up with a simple rule. In an [earlier issue](https://github.com/elixir-europe/rdmkit/issues/1130) perhaps we were thinking of uses like "metadata schema" or "database schema".  But in cases like "naming scheme" then "scheme" is perhaps the better word, or "[naming convention](https://en.wikipedia.org/wiki/Naming_convention)". 

The [style guide](https://rdmkit.elixir-europe.org/style_guide) does not talk about all possible uses of scheme/schema, but just says, 'Use “schema” if you mean a pattern for structuring data.' I don't know if we need to go into more details here. It doesn't appear to be a common word on the site. 

So my vote would be to:

- Adopt "naming convention" instead of "naming schema/scheme".
- Keep "identification scheme" (if that's what it's [conventionally called](https://en.wikipedia.org/wiki/Identification_scheme)), and if the readers of the page will understand it.
- Stick with "metadata schema".

**Comment by martin-nc** [comment]
@bianchini88 Yes - sorry, I wasn't paying enough attention to the actual sentence. It should definitely be plural. Should it be "identifier" or "identification"? On further (but not in-depth!) research, that seems more tricky to me now. They seem to be used interchangeably sometimes:

Screenshot 2023-02-09 at 17 10 17

"Identification scheme" seems to be used in cryptography and access control protocols (e.g. [this article](https://www.cambridge.org/core/books/abs/complexity-and-cryptography/identification-schemes/E64B768856B75A4AE06CCF042260E0D3)), but not really in the same sense.  I couldn't find anything on the FAIRsharing website. Maybe we should ask them? Other than that, and based on [this article](https://www.gov.uk/government/publications/linked-identifier-schemes-best-practice-guide/linked-identifier-schemes-best-practice-guide), I'd guess "identifier schemes" might be better. It's clearer that it's not about human identity, as well.

@nsoranzo yes, I was thinking the same. We should probably just remove "Schema" from the style guide, if everyone is happy with that. There isn't a straightforward way of covering all the uses of schema/scheme/convention etc.

**Comment by bedroesb** [comment]
@nsoranzo @bianchini88 @floradanna and @martin-nc I think I covered the feedback in my latest commit, thanks a lot for looking into this!

**Review by bianchini88 — APPROVED** [review]

**Pull request merged**